### DISCUSSION TITLE
Cefsharp browser print html to PDF, some PDF font mess

### DISCUSSION THREAD

**Discussion by rock117** [cefsharp/CefSharp#3676] [General]
Hi, I am using Cefsharp to export html to PDF using browser.PrintToPdfAsync, for some kind fonts(my company custom font), when print to pdf, the PDF font is mess, see below

![image](https://user-images.githubusercontent.com/2640825/124690410-bbe6d180-df0c-11eb-97c7-44d134757c31.png)

this also happen in chrome when using chrome Print function to save as PDF.

But Firefox's print PDF is OK
![image](https://user-images.githubusercontent.com/2640825/124690787-46c7cc00-df0d-11eb-939f-de7199f01711.png)

does anyone know this issue, and have any workaround?

Thanks

**Comment by rock117** [comment]
I checked the document property, in Frefox exported PDF, our font is show in the font list
![image](https://user-images.githubusercontent.com/2640825/124693005-3ade0900-df11-11eb-8446-20996174fb61.png)

But in Cefsharp exported PDF, the font list not contains our font

![image](https://user-images.githubusercontent.com/2640825/124693208-99a38280-df11-11eb-8e3e-50c6d85d029b.png)

**Comment by amaitland** [comment]
> this also happen in chrome when using chrome Print function to save as PDF.

Did you try checking the chromium issue tracker? Any issue that reproduces in Chrome that's the first place to start.

**Reply by rock117** [reply]
thanks amaitland, I raised a issue to chrome just now.

**Reply by amaitland** [reply]
You can execute JavaScript to change the font used, you might be able to get away with only changing the print media styles. Fallback to a known working font.
